src/samplers.jl

```diff
@@ -152,49 +152,40 @@ export NStepBatchSampler
 
 mutable struct NStepBatchSampler{traces}
```

Comment:
Does this need to be mutable?

Reply:
Nope, it wasn't the case before either actually.